build.gradle

```diff
@@ -62,8 +54,9 @@ subprojects {
             html.enabled = true
         }
     }
-    
+
     dependencies {
+        compile 'org.codehaus.groovy:groovy-all'
```

Comment:
Adding compile dependency to Groovy might make some users unhappy (not me obviously ;) ). Do you really need that?